policies/resource_cluster_policy.go

```diff
@@ -66,6 +101,9 @@ func ResourceClusterPolicy() *schema.Resource {
 			var request compute.EditPolicy
 			common.DataToStructPointer(d, s, &request)
 			request.PolicyId = d.Id()
+			if request.PolicyFamilyId != "" {
+				request.Definition = ""
```

Comment:
Why this? Is to fix previously updated definitions?

Reply:
The problem is the following:

1. When we override a cluster policy, then we need to use the `policy_override` field, instead of `definition`, otherwise we'll get an error.
2. When we read policy back after creation, this operation also brings back the `definition`
3. When we're trying to update an instance, then if we send the `definition` that we have in state, then we get an error saying that we need to use `policy_override` with cluster policy families.